README.md

```diff
@@ -21,26 +21,24 @@ yarn start
 
 ## Usage
 
-Point your device or webpage to your local machine and to send Snowplow events
-to be collected. Your IP address and listening port is listed at the bottom of
-the app. Events should appear within seconds.
+Point your device or webpage to your local machine and to send Snowplow events to be collected. Your
```

Comment:
While we're at it, the clause `and to send Snowplow events` in this sentence confuses me.

Reply:
Hah, reading it again has me confused too. I've changed it into something more sensible.